src/extensions/default/CodeFolding/main.js

```diff
@@ -348,5 +422,28 @@ define(function (require, exports, module) {
         }
     }
 
-    AppInit.appReady(init);
+    /**
+      * Register change listener for the preferences file.
+      */
+    function watchPrefsForChanges() {
+        prefs.prefBase.on("change", function (e, data) {
+            if (data.ids.indexOf("enabled") > -1) {
+                if (prefs.getSetting("enabled")) {
+                    init();
+                } else {
+                    var editor = EditorManager.getCurrentFullEditor();
+                    if (editor && editor._codeMirror && CodeMirror.commands.unfoldAll) {
+                        CodeMirror.commands.unfoldAll(editor._codeMirror);
+                        foldGutter.remove(editor._codeMirror);
+                    }
```

Comment:
Yes we would. Currently the other editors open 'catch up' when the active editor changes. I couldn't find an api for iterating through open editors in the working set. Is there one? How would you do it?